doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,258 @@
+* Remove `run_command` utility function by @portante in https://github.com/distributed-system-analysis/pbench/pull/2747
+* Add support for `--clients` to `pbench-linpack` by @portante in https://github.com/distributed-system-analysis/pbench/pull/2746
+* Move logic out of pbench-tool-meister-* files by @dbutenhof in https://github.com/distributed-system-analysis/pbench/pull/2741
+* Get rid of the `pbench-perl-JSON-XS` dependency (v0.71) by @ndokos in https://github.com/distributed-system-analysis/pbench/pull/2751
+* Tabs to spaces, isle 5 by @portante in https://github.com/distributed-system-analysis/pbench/pull/2750
+* Remove unused `common` directories by @portante in https://github.com/distributed-system-analysis/pbench/pull/2753
+* Add a big-hammer redis server cleanup by @portante in https://github.com/distributed-system-analysis/pbench/pull/2758
+* Remove `pbench-sysinfo-dump` reliance on `base` by @portante in https://github.com/distributed-system-analysis/pbench/pull/2754
+* Ensure `agent/base` is not used by Tool Meisters by @portante in https://github.com/distributed-system-analysis/pbench/pull/2755
+* Employ a `NamedTuple` for TDS parameters by @portante in https://github.com/distributed-system-analysis/pbench/pull/2756
+* Refactor Tool Meister start to consolidate logic by @portante in https://github.com/distributed-system-analysis/pbench/pull/2757
+* Simplify PID file names used by TM sub-system by @portante in https://github.com/distributed-system-analysis/pbench/pull/2759
+* Small changes deferred from recent PRs by @webbnh in https://github.com/distributed-system-analysis/pbench/pull/2761
+* Remove unused `bench-scripts` mocks by @portante in https://github.com/distributed-system-analysis/pbench/pull/2764
+* Correct misspelling in `bench-scripts/unittests` by @portante in https://github.com/distributed-system-analysis/pbench/pull/2767
+* Optional metadata should be optional by @portante in https://github.com/distributed-system-analysis/pbench/pull/2770
+* Support no tools registered by @portante in https://github.com/distributed-system-analysis/pbench/pull/2773
+* Fix unit testing for email_validator by @dbutenhof in https://github.com/distributed-system-analysis/pbench/pull/2776
+* Deprecate unused benchmark convenience scripts by @portante in https://github.com/distributed-system-analysis/pbench/pull/2780
+* Improved `linpack` benchmark wrapper by @portante in https://github.com/distributed-system-analysis/pbench/pull/2768
+* Remove support for trafficgen by @portante in https://github.com/distributed-system-analysis/pbench/pull/2781
+* Update use of `b0.71` CI image name for this branch by @portante in https://github.com/distributed-system-analysis/pbench/pull/2786
+
+
+**Full Changelog**: https://github.com/distributed-system-analysis/pbench/compare/v0.69-agent...v0.71.0-alpha.0
+
+
+v0.70.0 (Beta) Release Notes
```

Comment:
v0.71.0 ... there are 0.70 references scattered through the text that need to be updated.

Reply:
Fixed.